src/category_theory/preadditive/opposite.lean

```diff
@@ -27,4 +28,34 @@ instance : preadditive Cᵒᵖ :=
 @[simp] lemma unop_zero (X Y : Cᵒᵖ) : (0 : X ⟶ Y).unop = 0 := rfl
 @[simp] lemma unop_add {X Y : Cᵒᵖ} (f g : X ⟶ Y) : (f + g).unop = f.unop + g.unop := rfl
 
+variables {C} {D : Type*} [category D] [preadditive D]
+
+instance functor.op_additive (F : C ⥤ D) [F.additive] : F.op.additive := {}
+
+instance functor.right_op_additive (F : Cᵒᵖ ⥤ D) [F.additive] : F.right_op.additive :=
+{ map_zero' := λ X Y, begin
+    change (F.map 0).op = 0,
```

Comment:
`change` is a tactic I'm falling out of love with because Yakov pointed out that it can sometimes hide a really heavy `refl`. That's clearly not going on here but you could probably get rid of the change here and on line 42 with an appropriate lemma which you can rw with instead.